tools/build.gradle

```diff
@@ -0,0 +1,29 @@
+
+
+apply plugin: 'application'
+
+mainClassName = 'org.hl7.davinci.priorauth.tools.DatabaseTool'
```

Comment:
Does this limit us to only this one tool? If in the future we have another tool would it need to be in its own folder with a separate build file?

Reply:
This is setup to only build one tool per folder so maybe the folder should be named something else. I am not very good with gradle but was able to get something to work!